patching/plugin-commands-patching/src/patchCommit.ts

```diff
@@ -53,7 +64,7 @@ export async function handler (opts: install.InstallCommandOptions & Pick<Config
   } else if (!rootProjectManifest.pnpm.patchedDependencies) {
     rootProjectManifest.pnpm.patchedDependencies = {}
   }
-  rootProjectManifest.pnpm.patchedDependencies![pkgNameAndVersion] = `patches/${patchFileName}.patch`
+  rootProjectManifest.pnpm.patchedDependencies![pkgNameAndVersion] = `${patchesDirName}/${patchFileName}.patch`
```

Comment:
Slashes in `patchesDirName` should be normalized, so that `/` is only used.

Reply:
right, wrapped with `path.normalize`, thanks!